qiskit_ibm_experiment/client/local_client.py

```diff
@@ -0,0 +1,534 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Client for local Quantum experiment services."""
+
+import logging
+import os
+import uuid
+from typing import List, Dict, Optional, Union, Any
+import pandas as pd
+import numpy as np
+import json
+from qiskit_ibm_experiment.exceptions import IBMExperimentEntryNotFound, IBMExperimentEntryExists, IBMApiError
+
+logger = logging.getLogger(__name__)
+
+class LocalExperimentClient():
```

Comment:
I think it should inherit from `ExperimentClient`

Reply:
They are distinct classes with no shared functionality. It might make sense to have an abstract interface they both inherit from.